[PATCH] vp8: drop buggy libvpx legacy handling

Legacy handling for older libvpx versions were added back in 2010, but this
was unfortunately buggy as these symbols are enums and not defines - So they
are never defined and the legacy symbols always used.

Now that the legacy symbols are completely removed from libvpx from v1.4.0
onwards this breaks.  Fix it by simply dropping the legacy handling, nobody
should be using libvpx versions from before 2010 (or gstreamer 0.10 for that
matter) any more.

Signed-off-by: Peter Korsgaard <peter@korsgaard.com>
---
 ext/vp8/gstvp8utils.h |   17 -----------------
 1 file changed, 17 deletions(-)

Index: gst-plugins-bad-0.10.23/ext/vp8/gstvp8utils.h
===================================================================
--- gst-plugins-bad-0.10.23.orig/ext/vp8/gstvp8utils.h
+++ gst-plugins-bad-0.10.23/ext/vp8/gstvp8utils.h
@@ -25,23 +25,6 @@
 
 G_BEGIN_DECLS
 
-/* Some compatibility defines for older libvpx versions */
-#ifndef VPX_IMG_FMT_I420
-#define VPX_IMG_FMT_I420 IMG_FMT_I420
-#endif
-
-#ifndef VPX_PLANE_Y
-#define VPX_PLANE_Y PLANE_Y
-#endif
-
-#ifndef VPX_PLANE_U
-#define VPX_PLANE_U PLANE_U
-#endif
-
-#ifndef VPX_PLANE_V
-#define VPX_PLANE_V PLANE_V
-#endif
-
 const char * gst_vpx_error_name (vpx_codec_err_t status);
 
 G_END_DECLS
